tools/kmod/sof_remove.sh

```diff
@@ -120,6 +120,7 @@ remove_module snd_soc_hda_codec
 # platform drivers
 #-------------------------------------------
 remove_module snd_sof_intel_hda_generic
+remove_module snd_sof_intel_hda_common
```

Comment:
I am pretty sure this does not work if you have snd_sof_intel_hda_sdw_bpt in use.

See https://github.com/thesofproject/linux/pull/4679/commits/af779a3ca9bae844fb1691caaba7243e49afbe71#diff-117faa254cc1264b915358fff9c601a8b15e7545c3, the BPT stuff depends on the hda_common stuff.

so if you want to deal with older kernels, you can add this line and ignore errors, and keep it line 136 where it was.